example/test.re

```diff
@@ -10,6 +10,7 @@ let arialNarrow =
       ~fontFamily="Arial FontFace Test",
       ~src=[localUrl("Arial Narrow")],
       ~fontStyle=normal,
+      ~fontWeight=500,
```

Comment:
This named arg just wasn't set in the original version. Because I rolled my own "font face" method off of emotion, I made sure weight was being propagated through.